any_member.hpp

```diff
@@ -400,6 +400,16 @@ any_member<ClassType>& any_member<ClassType>::assign(std::string const& s)
     return *this;
 }
 
+/// The standard implementation of swap() doesn't work for any_member<> as it
```

Comment:
I'm not sure if this is still true in C++11.

Also, I don't think we actually need a global `swap()` any more, do we? I.e. this commit could be simple dropped.

Finally, if you decide to keep it for some reason, there is a typo in the commit message:

> hence we this swap() function()

should be just

> hence this swap() function